### ISSUE TITLE
gets info from route

### ISSUE THREAD

**Issue by kangkyu** [LARailsLearners/box-office#2]
open `localhost:4567/tt0078346` then the page says "Superman"

**Comment by thomasjinlo** [comment]
Does anyone need the API key for TMDb?

**Comment by kangkyu** [comment]
don't we? it's in your code, as `ENV` and you need yours if run it locally for development

**Comment by thomasjinlo** [comment]
I meant I can provide my API key if people don't want to sign up for a new one

**Comment by kangkyu** [comment]
But still, it's public repo so we better not include your own API key to readme or in the code. Still we need some instruction how to develop and have value on `ENV['TMDB_API_KEY']` I think...

**Comment by server-monitor** [comment]
API keys are usually stored on some env manager. Each developer usually has
his own set of keys then the keys are loaded using env manager. I usually
use Figaro in Rails, then my keys and other data I don't want committed to
the repo are stored in config/application.yml

**Comment by kangkyu** [comment]
yes that's better, then you can do that in your commit of this PR #14  ✨

**Comment by server-monitor** [comment]
I'm going to do some research first on how to integrate Figaro into Sinatra. It shouldn't be too bad.

**Comment by kangkyu** [comment]
Did you close this issue because we're not implementing this any more @btmash ? Come to our slack chat https://laruby.slack.com/messages/eastside_study_group/

**Comment by kangkyu** [comment]
@thomasjinlo @server-monitor  now I see there's open issue https://github.com/LARailsLearners/box-office/issues/11 and we better add dotenv (or figaro) as a separate PR. Anyways we can finish this issue first without it

**Comment by server-monitor** [comment]
I did a little research last night and it seems dotenv is easier to integrate with Sinatra than Figaro so we'll go with dotenv if there are no objectsions
